benchmark/king2dcompressible/ala/code.cc

```diff
@@ -0,0 +1,284 @@
+
+
+#include <aspect/postprocess/interface.h>
+#include <aspect/material_model/simple.h>
+#include <aspect/boundary_temperature/interface.h>
+#include <aspect/simulator_access.h>
+#include <aspect/global.h>
+#include <aspect/geometry_model/box.h>
+
+#include <deal.II/dofs/dof_tools.h>
+#include <deal.II/numerics/data_out.h>
+#include <deal.II/base/quadrature_lib.h>
+#include <deal.II/base/function_lib.h>
+#include <deal.II/numerics/error_estimator.h>
+#include <deal.II/numerics/vector_tools.h>
+
+
+
+namespace aspect
+{
+  using namespace dealii;
+
+
+
+  /**
+   * This benchmark is from the article
+   * @code
+   * @endcode
```

Comment:
Make same changes as in the other `code.cc` files. In fact, you'd think that there ought to be ways to reduce code duplication. I haven't looked through the files in detail, but it seems like there is a significant amount of common code, right?